Book.java

```diff
@@ -0,0 +1,32 @@
+public class Book {
+    Book(int initialNoOfBook){
+        this.noOfCopies = initialNoOfBook;
+    }
+    private int noOfCopies ;
+   void setNoOfCopies(int number){  // seters
+       if(number>0){
+       this.noOfCopies = number;
+       }
+   }
+   int setNoOfCopies(){ //getter
+     return this.noOfCopies;
+    }
+    public void increaseNo(int howMuch){  // method to increase no of book
+        setNoOfCopies(this.noOfCopies+howMuch);
```

Comment:
```suggestion
        setNoOfCopies(this.noOfCopies + howMuch);
```
There should be space around all the operators